HerPublicWebsite.UnitTests/BusinessLogic/ExternalServices/EpbEpc/EpbEpcApiTests.cs

```diff
@@ -41,25 +42,10 @@ public void Setup()
         HttpRequestHelper.handler = mockHttpHandler;
     }
 
-    private readonly EpcAssessmentDto correctAssessment = new()
-    {
-        Address = new EpcAddressDto
-        {
-            AddressLine1 = "22 Acacia Avenue",
-            AddressLine2 = "Upper Wellgood",
-            AddressLine3 = "A Building",
-            AddressLine4 = "A Place",
-            Town = "Fulchester",
-            Postcode = "FL23 4JA"
-        },
-        Uprn = "001234567890",
-        LodgementDate = new DateTime(2020, 2, 29),
-        CurrentBand = EpcRating.D
-    };
-
     [Test]
     public async Task GetEpcFromUpn()
```

Comment:
I know this isn't part of this review, but I can't tell what this test is doing easily from the name

Reply:
done